documenting.rst

```diff
@@ -456,7 +457,7 @@ the extension mechanisms of reST, and Sphinx makes heavy use of it.
 
 Basically, a directive consists of a name, arguments, options and content. (Keep
 this terminology in mind, it is used in the next chapter describing custom
-directives.)  Looking at this example, ::
+directives.)  Looking at this example,::
```

Comment:
Isn't the space necessary here?
You could remove the comma and use ``example::``, but otherwise I would leave it as is.

Reply:
That's a bit of Eclipse formatting (good for some things), not intentional. I see that in the original, the comma shows and the colon is invisible. I'll replace with the long form.